packages/fluentbit/extra/dcos-fluentbit.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Fluentbit: collects and forwards logs
```

Comment:
nit: "Fluent Bit" seems to be the preferred human-readable formatting, judging from docs.